data/named-accounts.ts

```diff
@@ -1,73 +1,182 @@
+        ...getAddress(base, ZERO_ADDRESS),
+        ...getAddress(arbitrum, '0x2f2a2543B76A4166549F7aaB2e75Bef0aefC5B0f'),
+        ...getAddress(mantle, '0xCAbAE6f6Ea1ecaB08Ad02fE02ce9A44F09aebfA2')
     },
     bnt: {
-        ...mainnet('0x1F573D6Fb3F13d689FF844B4cE37794d79a7FF1C')
+        ...getAddress(mainnet, '0x1F573D6Fb3F13d689FF844B4cE37794d79a7FF1C'),
+        ...getAddress(base, ZERO_ADDRESS),
+        ...getAddress(arbitrum, ZERO_ADDRESS),
+        ...getAddress(mantle, ZERO_ADDRESS)
     },
     link: {
-        ...mainnet('0x514910771AF9Ca656af840dff83E8264EcF986CA')
+        ...getAddress(mainnet, '0x514910771AF9Ca656af840dff83E8264EcF986CA'),
+        ...getAddress(base, ZERO_ADDRESS),
+        ...getAddress(arbitrum, ZERO_ADDRESS),
+        ...getAddress(mantle, ZERO_ADDRESS)
     },
 };
 
 const BancorNamedAccounts = {
-    bancorNetworkV3: { ...mainnet('0xeEF417e1D5CC832e619ae18D2F140De2999dD4fB') }
+    bancorNetworkV3: { 
+        ...getAddress(mainnet, '0xeEF417e1D5CC832e619ae18D2F140De2999dD4fB'),
+        ...getAddress(mantle, ZERO_ADDRESS),
+        ...getAddress(base, ZERO_ADDRESS),
+        ...getAddress(arbitrum, ZERO_ADDRESS)
+    }
 };
 
+function getAddress(func: (arg: string) => object | undefined, arg: string): object {
+    const result = func(arg);
+    return result || {};
+}
+
 export const NamedAccounts = {
     deployer: {
-        ...mainnet('ledger://0x5bEBA4D3533a963Dedb270a95ae5f7752fA0Fe22'),
-        ...rinkeby('ledger://0x0f28D58c00F9373C00811E9576eE803B4eF98abe')
+        ...getAddress(mainnet, 'ledger://0x5bEBA4D3533a963Dedb270a95ae5f7752fA0Fe22'),
+        ...getAddress(sepolia, 'ledger://0x0f28D58c00F9373C00811E9576eE803B4eF98abe'),
+        ...getAddress(base, 'ledger://0x0f28D58c00F9373C00811E9576eE803B4eF98abe'),
+        ...getAddress(arbitrum, 'ledger://0x0f28D58c00F9373C00811E9576eE803B4eF98abe'),
+        ...getAddress(mantle, 'ledger://0x5bEBA4D3533a963Dedb270a95ae5f7752fA0Fe22'),
     },
-    deployerV2: { ...mainnet('0xdfeE8DC240c6CadC2c7f7f9c257c259914dEa84E') },
-    foundationMultisig: { ...mainnet('0xeBeD45Ca22fcF70AdCcAb7618C51A3Dbb06C8d83') },
-    foundationMultisig2: { ...mainnet('0x0c333d48Af19c2b42577f3C8f4779F0347F8C819') },
-    daoMultisig: { ...mainnet('0x7e3692a6d8c34a762079fa9057aed87be7e67cb8') },
-    daoPauserMultisig: { ...mainnet('0xc140c1CD2e587fC04DAa780d49b616F768476660') },
+    deployerV2: { ...getAddress(mainnet, '0xdfeE8DC240c6CadC2c7f7f9c257c259914dEa84E') },
+    foundationMultisig: { ...getAddress(mainnet, '0xeBeD45Ca22fcF70AdCcAb7618C51A3Dbb06C8d83') },
+    foundationMultisig2: { ...getAddress(mainnet, '0x0c333d48Af19c2b42577f3C8f4779F0347F8C819') },
+    daoMultisig: { ...getAddress(mainnet, '0x7e3692a6d8c34a762079fa9057aed87be7e67cb8') },
+    daoPauserMultisig: { ...getAddress(mainnet, '0xc140c1CD2e587fC04DAa780d49b616F768476660') },
 
     ...TokenNamedAccounts,
     ...TestNamedAccounts,
     ...BancorNamedAccounts
-};
+};
```

Comment:
Missing new line at the end of the while. You should integrate/run the formatter.

Reply:
fixed